esmvaltool/cmorizers/obs/cmorize_obs_era_interim.py

```diff
@@ -0,0 +1,329 @@
+            if not cube.coords('day_of_year'):
+                iris.coord_categorisation.add_day_of_year(cube, 'time')
+            if not cube.coords('year'):
+                iris.coord_categorisation.add_year(cube, 'time')
+            cube = cube.aggregated_by(['day_of_year', 'year'],
+                                      iris.analysis.SUM)
+        else:
+            cube = daily_statistics(cube, 'mean')
+        # Remove daily statistics helpers
+        cube.remove_coord(cube.coord('day_of_year'))
+        cube.remove_coord(cube.coord('year'))
+        # Correct the time bound
+        cube.coord('time').points = cube.coord('time').units.date2num(
+            [
+                cell.point.replace(hour=12, minute=0, second=0,
+                                   microsecond=0)
+                for cell in cube.coord('time').cells()
+            ]
+        )
+        cube.coord('time').bounds = None
+        cube.coord('time').guess_bounds()
+    return cube
+
+
+def _get_files(in_dir, var):
+    # Make a dictionary with the keys that are years
+    files_dict = defaultdict(list)
+    for in_file in var['files']:
+        files_lst = sorted(list(Path(in_dir).glob(in_file)))
+        for item in files_lst:
+            key = str(item.stem).split('_')[-1]
+            files_dict[key].append(item)
+    # Check if files are complete
+    for key in files_dict.copy():
+        if len(files_dict[key]) != len(var['files']):
+            logger.info("CMORizing %s at time '%s' needs '%s' input file/s",
+                        var['short_name'], key, len(var['files']))
+            files_dict.pop(key)
+    return files_dict
+
+
+def _extract_variable(in_file, var, cfg, out_dir):
+    if 'files' in var:
+        logger.info("CMORizing variable '%s' from input file/s '%s'",
+                    var['short_name'], [str(item) for item in in_file])
+    else:
+        logger.info("CMORizing variable '%s' from input file/s '%s'",
+                    var['short_name'], in_file)
+    attributes = deepcopy(cfg['attributes'])
+    attributes['mip'] = var['mip']
+    cmor_table = CMOR_TABLES[attributes['project_id']]
+    definition = cmor_table.get_variable(var['mip'], var['short_name'])
+
+    with catch_warnings():
+        filterwarnings(
+            action='ignore',
+            message="Ignoring netCDF variable 'tcc' invalid units '(0 - 1)'",
+            category=UserWarning,
+            module='iris',
+        )
+        filterwarnings(
+            action='ignore',
+            message="Ignoring netCDF variable 'lsm' invalid units '(0 - 1)'",
+            category=UserWarning,
+            module='iris',
+        )
+        filterwarnings(
+            action='ignore',
+            message=("Ignoring netCDF variable 'e' invalid units "
+                     "'m of water equivalent'"),
+            category=UserWarning,
+            module='iris',
+        )
+        if 'files' in var:
```

Comment:
There is no need for this check, since you only specify `operator: sum` when there are more than one input variable/file

Reply:
@bouweandela thanks. This check is needed because the key "operator" (in ERA-Interim.yaml) only appears in the multiple case.